### PR TITLE
prepare v0.9.6

### PR DESCRIPTION
**Pull request by eeeebbbbrrrr**
This is an "out of band" patch release for a v0.9.6, based directly off v0.9.5.  This release fixes a bug with `Datum` to `Date` conversions.  Unfortunately, this bug essentially manifests itself as a Y2K bug where dates prior to 2000-01-01, when used in an `Array<Date>` fail to convert.

When updating, please make sure to `cargo install cargo-pgrx --locked` and update your crate dependencies to v0.9.6.

## What's Changed

- PR #1177: Cast Date Datums to `pg_sys::DateADT` (`i32`) instead of `isize`)

### CODE REVIEW

**Comment by eeeebbbbrrrr** [comment]
I'm just going to regular-merge this straight into develp

File: pgrx/src/datum/date.rs [hunk 1]
```diff
@@ -63,7 +63,7 @@ impl TryFrom<pg_sys::Datum> for Date {
 
     #[inline]
     fn try_from(datum: pg_sys::Datum) -> Result<Self, Self::Error> {
-        pg_sys::DateADT::try_from(datum.value() as isize).map(|d| Date(d))
+        Ok(Date(datum.value() as pg_sys::DateADT))
```
Comment (eeeebbbbrrrr) [inline pgrx/src/datum/date.rs:66]: This is the reason for this release.

**Review by workingjubilee — APPROVED** [review]
Apparently Ubuntu repos are unavailable but this is probably good if macOS CI passes.

**Comment by eeeebbbbrrrr** [comment]
Also passed locally on... wait for it... Ubuntu!

**Pull request merged**